src/iop/denoiseprofile.c

```diff
@@ -2705,11 +2797,15 @@ void gui_init(dt_iop_module_t *self)
                                          "wavelets work best for `color' blending"));
   gtk_widget_set_tooltip_text(g->radius, _("radius of the patches to match. increase for more sharpness"));
   gtk_widget_set_tooltip_text(g->nbhood, _("emergency use only: radius of the neighbourhood to search patches in. increase for better denoising performance, but watch the long runtimes! large radii can be very slow. you have been warned"));
+  gtk_widget_set_tooltip_text(g->scattering,
+                              _("scattering of the neighbourhood to search patches in. increase for better "
+                                "coarse-grain noise reduction. does not affect execution time."));
```

Comment:
I just notice that... "does not affect execution time", using this slider I had quickly a large slowdown with values like 15. Is this comment still valid?

Reply:
Forget this question, I was using another slider! This one indeed does not increase the execution time.

Reply:
No problem :-)